systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.strimzi.test.ClusterController;
+import io.strimzi.test.KafkaCluster;
+import io.strimzi.test.Namespace;
+import io.strimzi.test.OpenShiftOnly;
+import io.strimzi.test.Resources;
+import io.strimzi.test.StrimziRunner;
+import io.strimzi.test.TestUtils;
+import io.strimzi.test.k8s.KubeClient;
+import io.strimzi.test.k8s.KubeClusterException;
+import io.strimzi.test.k8s.KubeClusterResource;
+import io.strimzi.test.k8s.Oc;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.regex.Pattern;
+
+import static io.strimzi.test.TestUtils.map;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(StrimziRunner.class)
+@Namespace(KafkaClusterTest.NAMESPACE)
+//@Resources(value = "../examples/install/cluster-controller", asAdmin = true)
```

Comment:
Could this be deleted? Or does it have some meaning?

Reply:
I'll be pushing a commit to remove the commented out annotation